autocorpus/run.py

```diff
@@ -16,11 +16,11 @@ def run_autocorpus(config, structure, key, output_format):
     """
     ac = Autocorpus(
         config=config,
-        main_text=structure[key]["main_text"],
+        main_text=Path(structure[key]["main_text"]),
```

Comment:
This is going to cause some conflicts with my branch because I noticed the inconsistency with `str` object being labelled as `Path`, but I changed them all to be `str` instead of making this change to turn it into a `Path`.

I prefer that it be a `Path`, but it is going to be a challenge. Can you swap it around?